inference-engine/src/vpu/common/src/ngraph/transformations/dynamic_to_static_shape_transpose.cpp

```diff
@@ -15,18 +15,18 @@
 namespace vpu {
 
 void dynamicToStaticShapeTranspose(std::shared_ptr<ngraph::Node> target) {
-    const auto dsr = target->get_argument(0);
+    const auto dsr = target->input_value(0).get_node_shared_ptr();
     VPU_THROW_UNLESS(ngraph::as_type_ptr<ngraph::vpu::op::DynamicShapeResolver>(dsr),
         "DynamicToStaticShape transformation for {} of type {} expects {} as input with index {}",
         target->get_friendly_name(), target->get_type_info(), ngraph::vpu::op::DynamicShapeResolver::type_info, 0);
 
-    const auto transposition = target->get_argument(1);
+    const auto transposition = target->input_value(1).get_node_shared_ptr();
     VPU_THROW_UNLESS(ngraph::as_type_ptr<ngraph::opset3::Constant>(transposition),
         "DynamicToStaticShape transformation for {] of type {} expects {} as input with index {}",
         target->get_friendly_name(), target->get_type_info(), ngraph::opset3::Constant::type_info, 1);
 
     const auto transpose = std::dynamic_pointer_cast<ngraph::opset3::Transpose>(target);
-    const auto copied = transpose->copy_with_new_args(target->get_arguments());
+    const auto copied = transpose->clone_with_new_inputs(target->input_values());
```

Comment:
Why you changed `copy_with_new_args` with `clone_with_new_inputs`? Is it expected?

Reply:
It's actually correct, `copy_with_new_args` is now deprecated and should be replaced with `clone_with_new_inputs`.